Kitodo/src/main/java/org/kitodo/production/forms/createprocess/CreateProcessForm.java

```diff
@@ -651,4 +623,74 @@ public boolean metadataAddableToGroup(TreeNode metadataNode) {
     public void prepareAddableMetadataForGroup(TreeNode treeNode) {
         addMetadataDialog.prepareAddableMetadataForGroup(getMainProcess().getRuleset(), treeNode);
     }
+
+    /**
+     * Get process ancestors.
+     *
+     * @return process ancestors
+     */
+    public List<TempProcess> getProcessAncestors() {
+        return this.processes;
+    }
+
+    /**
+     * Get process children.
+     *
+     * @return process children
+     */
+    public List<TempProcess> getProcessChildren() {
+        return this.childProcesses;
+    }
+
+    /**
+     * Get ID metadata.
+     *
+     * @param tempProcess TempProcess whose ID metadata is returned
+     * @return ID metadata
+     */
+    public String getID(TempProcess tempProcess) {
```

Comment:
I would expect to get an database id here (I know, its a tempProcess, but still... ) Can you be more specific in the methodname?

Reply:
This method returns the catalog ID which is used to query individual records from the search interface. It is configured in the `kitodo_opac.xml` using the `identifierMetadata` element. This metadata is displayed in the import mask on the buttons in the process hierarchy to identify the individual records imported.

Reply:
Is the name `getCatalogueId` not a better name then `getMetadataId`?